src/tiledb/cloud/asset.py

```diff
@@ -19,8 +20,8 @@
 
 
 def list(
+    teamspace: str,
```

Comment:
Temporarily, teamspace is a required positional parameter. It's like this in the Server API.